package.json

```diff
@@ -38,16 +40,7 @@
   "engine": {
     "node": ">=0.8.x"
   },
-  "licenses": [
-    {
-      "type": "GPL",
-      "url": "https://github.com/wikimedia/jquery.i18n/blob/master/GPL-LICENSE"
-    },
-    {
-      "type": "MIT",
-      "url": "https://github.com/wikimedia/jquery.i18n/blob/master/MIT-LICENSE"
-    }
-  ],
+  "license": "(MIT OR GPL-2.0)",
```

Comment:
The library didn't have anything saying "or later" in it, so I took the safer option. Trivial to fix if it was GPL2+ all the time.